MagazineLayout/Public/MagazineLayout.swift

```diff
@@ -490,7 +496,7 @@ public final class MagazineLayout: UICollectionViewLayout {
       itemLayoutAttributesForPendingAnimations[finalIndexPath] = attributes
       return attributes
     } else {
-      return super.layoutAttributesForItem(at: itemIndexPath)
```

Comment:
noticed this was the wrong super call

Reply:
oh wow, were these ever getting hit though?

Reply:
Yeah I haven't seen it hit